plugins/woocommerce-admin/client/task-lists/progress-header/progress-header.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { useMemo } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { ONBOARDING_STORE_NAME, TaskListType } from '@woocommerce/data';
+import { getSetting } from '@woocommerce/settings';
+
+/**
+ * Internal dependencies
+ */
+import './progress-header.scss';
+import { TaskListMenu } from '~/tasks/task-list-menu';
+
+type ProgressHeaderProps = {
+	taskListId: string;
+};
+
+export const ProgressHeader: React.FC< ProgressHeaderProps > = ( {
+	taskListId,
+} ) => {
+	const { loading, tasksCount, completedCount, hasVisitedTasks } = useSelect(
+		( select ) => {
+			const isResolving = select( ONBOARDING_STORE_NAME ).isResolving(
```

Comment:
This is working well but I wonder if using `hasFinishedResolution` instead of `isResolving` would be better here (p90Yrv-1Yq-p2).